src/IAM/Infrastructure/Persistence/MySQL/Logout/DbalRefreshTokenStore.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Twitter\IAM\Infrastructure\Persistence\MySQL\Logout;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\ParameterType;
+use Twitter\IAM\Application\Logout\RefreshTokenStoreInterface;
+use Twitter\IAM\Domain\Auth\Exception\AuthTokenInvalidException;
+use Twitter\IAM\Infrastructure\Persistence\Auth\RefreshTokenHasher;
+use Twitter\Shared\Infrastructure\Persistence\UuidBinaryConverter;
+
+final class DbalRefreshTokenStore implements RefreshTokenStoreInterface
```

Comment:
As specified in the namespace, this implementation is for MySQL. Therefore, specify MySQL in the class name. For example:

```suggestion
final class MySQLRefreshTokenStore implements RefreshTokenStoreInterface
```